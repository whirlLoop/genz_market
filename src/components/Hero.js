import React from "react";
import { Grid, Container, Button, Typography, Box } from "@mui/material";
import WorkDoodle from "../assets/work-doodle-icon.jpg";
import MicrosoftIcon from "../assets/logos/microsoft.svg";
import GodaddyIcon from "../assets/logos/godaddy.svg";

export default function Landing() {
	return (
		<Container>
			<Grid
				container
				rowSpacing={1}
				columnSpacing={{ xs: 1, sm: 2, md: 4 }}
				sx={{ display: "flex", alignItems: "center" }}
				p
			>
				<Grid item xs={7}>
					<Typography variant="h2" sx={{ fontSize: "3rem" }}>
						Freedom to express your creative self
					</Typography>
					<Typography variant="p" sx={{ fontSize: "2rem", lineHeight: "2rem" }}>
						Forget the old rules. You can have the best people. Right now. Right
						here.
					</Typography>
					<Box sx={{ marginTop: "50px" }}>
						<Button variant="contained" sx={{ marginRight: "10px" }}>
							Find Work
						</Button>
						<Button variant="outlined">Find Talent</Button>
					</Box>
					<Box sx={{ marginTop: "50px" }}>
						<Typography>Trusted by</Typography>
						<Box
							sx={{ marginTop: "10px", display: "flex", alignItems: "center" }}
						>
							<img
								src={MicrosoftIcon}
								alt="Work Doodle"
								style={{ width: "auto", height: "35px", marginRight: "20px" }}
							/>
							<img
								src={GodaddyIcon}
								alt="Work Doodle"
								style={{ width: "auto", height: "25px" }}
							/>
						</Box>
					</Box>
				</Grid>
				<Grid item xs={5}>
					<img
						src={WorkDoodle}
						alt="Work Doodle"
						style={{ width: "100%", height: "300px", borderRadius: "50%" }}
					/>
				</Grid>
			</Grid>
		</Container>
	);
}
