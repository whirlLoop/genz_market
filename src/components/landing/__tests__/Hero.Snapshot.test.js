import React from "react";
import Hero from "../Hero";
import renderer from "react-test-renderer";

it("Renders correctly", () => {
	const tree = renderer.create(<Hero onMessageSubmit={jest.fn()} />);
	expect(tree).toMatchSnapshot();
});
