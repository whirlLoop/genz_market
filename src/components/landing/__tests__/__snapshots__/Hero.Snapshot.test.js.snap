// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  data-testid="landing-component"
>
  <div
    className="MuiGrid-root MuiGrid-container css-uk0h20-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7 css-uqwprf-MuiGrid-root"
    >
      <h2
        className="MuiTypography-root MuiTypography-h2 css-689ebp-MuiTypography-root"
      >
        Freedom to express your creative self
      </h2>
      <span
        className="MuiTypography-root MuiTypography-p css-1kbix5p-MuiTypography-root"
      >
        Forget the old rules. You can have the best people. Right now. Right here.
      </span>
      <div
        className="MuiBox-root css-jn01mi"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-di5245-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Find Work
        </button>
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Find Talent
        </button>
      </div>
      <div
        className="MuiBox-root css-jn01mi"
      >
        <p
          className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Trusted by
        </p>
        <div
          className="MuiBox-root css-1p2zrxa"
        >
          <img
            alt="Microsoft Icon"
            src="test-file-stub"
            style={
              {
                "height": "35px",
                "marginRight": "20px",
                "width": "auto",
              }
            }
          />
          <img
            alt="Godaddy Icon"
            src="test-file-stub"
            style={
              {
                "height": "25px",
                "width": "auto",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-k3f0g4-MuiGrid-root"
    >
      <img
        alt="Work Doodle"
        src="test-file-stub"
        style={
          {
            "borderRadius": "50%",
            "height": "300px",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;
